hack/monitor_prow.py

```diff
@@ -17,16 +17,26 @@
 RESET = exec_cmd('tput', 'sgr0')
 CLEAR = exec_cmd('tput', 'clear')
 
+
+def run_oc(args):
+    command = ['oc', '--loglevel', '10', '--namespace', 'ci']
+    command.extend(args)
+    process = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    stdout, stderr = process.communicate()
+    if process.returncode != 0:
+        print(stderr.decode('utf-8'))
+        raise subprocess.CalledProcessError(returncode=process.returncode, cmd=command, output=stdout)
+
+    return stdout.decode('utf-8')
```

Comment:
What's wrong with little `exec_cmd` over there?  ^

Reply:
doesn't grab stderr in a separate thign

Reply:
(which is how you get loglevel output from not garbling your JSON etc)

Reply:
Does `oc logs` combine `stdout` and `stderr` into its `stdout`?  Or will the `--loglevel 10` output get mixed with the actual logs?

Reply:
All logger output is `stderr`